applications/nrf_desktop/README.rst

```diff
@@ -1556,6 +1556,8 @@ The B0 bootloader requires the following options enabled:
 * :kconfig:option:`CONFIG_SB_SIGNING_KEY_FILE` - Required for providing the signature used for image signing and verification.
 * :kconfig:option:`CONFIG_FW_INFO` - Required for the application versioning information.
 * :kconfig:option:`CONFIG_FW_INFO_FIRMWARE_VERSION` - Enable this option to set the version of the application after you enabled :kconfig:option:`CONFIG_FW_INFO`.
+  For the DFU package generation, you need to update this configuration in the main application image and the ``s1_image`` child image.
+  You can do that by rebuilding the application from scratch or by changing the configuration independently in both images with the help of menuconfig.
```

Comment:
If I update image version in application's `prj.conf`, version for both images is updated.
If I update through menuconfig, `manifest.json` in the zip shows both images with updated versions, but the actual firmware version included in the S1 image is not updated. It seems like a bug. Maybe we need to resolve it before introducing documentation.